omod/src/main/java/org/openmrs/module/owa/web/controller/AddAppController.java

```diff
@@ -74,7 +74,8 @@ public String upload(@RequestParam("file") MultipartFile file, HttpServletReques
                     message = messageSourceService.getMessage("owa.not_a_zip");
                     log.warn("App is not a zip archive");
                     uploadedFile.delete();
-                    return message;
+                    model.addAttribute("message", "App is not a zip archive");
```

Comment:
we do not want to load our messages from code. It should be taken from the messages.properties, like it was done in L#74 using messageSourceService